zookeeper-server/src/main/java/org/apache/zookeeper/server/auth/znode/groupacl/ZNodeGroupAclProperties.java

```diff
@@ -32,6 +33,10 @@
   private static ZNodeGroupAclProperties instance = null;
 
   private ZNodeGroupAclProperties() {
+    serverDedicatedDomain = System.getProperty(DEDICATED_DOMAIN);
+    if (serverDedicatedDomain != null && !serverDedicatedDomain.isEmpty()) {
+      isConnectionFilteringEnabled = true;
```

Comment:
Why do we need to create a separate boolean `isConnectionFilteringEnabled`? This is not something that user can set, right?

Reply:
Please see my above reply